### ISSUE TITLE
Debugtoolbar-Injected Code not compatible for XML

### ISSUE THREAD

**Issue by HolgerPeters** [Pylons/pyramid_debugtoolbar#359]
Debugtoolbar seems to inject this code into my page

```
<link rel="stylesheet" type="text/css" href="http://127.0.0.1:56322/_debug_toolbar/static/toolbar/toolbar_button.css">

<div id="pDebug">
    <div  id="pDebugToolbarHandle">
        <a title="Show Toolbar" id="pShowToolBarButton"
           href="http://127.0.0.1:56322/_debug_toolbar/313430303338303032343633313230" target="pDebugToolbar">&#171;</a>
    </div>
</div>
```

When I set the resulting `ContentType` to ` self.request.response.content_type = 'application/xhtml+xml'`, this leads to an error in Firefox's XHTML parser. (I can of course rely on Firefox's HTML5 parser to be accepting of XML syntax, but I think it would be nice to have a configuraiton option for the debug toolbar to inject XML properly).

**Comment by mmerickel** [comment]
Would you mind submitting a PR that injects the correct syntax in that scenario? I do not see the need for a configuration option. The toolbar knows the content type and could adjust the syntax appropriately.

**Comment by mmerickel** [comment]
fixed via #360
